frontend/public/components/start-guide.tsx

```diff
@@ -8,10 +8,15 @@ import { FLAGS } from '@console/shared/src/constants';
 import { createProjectMessageStateToProps } from '../reducers/ui';
 import { Disabled, HintBlock, ExternalLink, openshiftHelpBase, LinkifyExternal } from './utils';
 import { connectToFlags } from '../reducers/features';
-import { ProjectModel, RoleModel, StorageClassModel } from '../models';
+import { ProjectModel, RoleModel, StorageClassModel, PersistentVolumeModel } from '../models';
 import { createProjectModal } from './modals/create-namespace-modal';
 
-const WHITELIST = [RoleModel.kind, StorageClassModel.kind];
+const WHITELIST = [RoleModel.kind, PersistentVolumeModel.kind];
+const CLUSTER_WIDE_WHITELIST = [
```

Comment:
Not new, but If we still need these globals in the final solution, we should probably rename them to use better language (remove "whitelist" terminology).